tests/common-utils.js

```diff
@@ -14,15 +14,15 @@ commonUtils.params = function () {
   if (commonUtils.isNode()) {
     return process.env;
   }
-  var paramStr = document.location.search.slice(1);
-  return paramStr.split('&').reduce(function (acc, val) {
-    if (!val) {
-      return acc;
-    }
-    var tmp = val.split('=');
-    acc[tmp[0]] = decodeURIComponent(tmp[1]) || true;
-    return acc;
-  }, {});
+  const usp = new URLSearchParams(window.location.search);
+  const acc = {};
```

Comment:
Let's rename `acc` as it doesn't make sense now that this isn't a reduce. Just `params` or `result` would be fine.